app/views/citizens/additional_accounts/index.html.erb

```diff
@@ -2,14 +2,15 @@
   <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
 <% end %>
 
+  <% content_for(:page_title) { t('.field_set_header') } %>
+
   <%= form_tag(citizens_additional_accounts_path) do %>
 
     <%= govuk_form_group(
           show_error_if: @error,
           hint_id: :additional_account_hint,
           input: :additional_account
         ) do %>
-      <%= govuk_fieldset_header t('.field_set_header'), padding_below: 6 %>
```

Comment:
You still need this line - but with the `content_for`
```erb
<%= govuk_fieldset_header content_for(:page_title), padding_below: 6 %>
```
To clarify - most pages will needs two instances of `content_for` - One (at the top of the page) sets the value so that it can be used withing the page and in the HTML title), and a second one within the markup, that uses the value set in the other call.